src/components/EventSummary.js

```diff
@@ -1,16 +1,22 @@
-import React, { Fragment } from 'react'
-import { Header, Segment, Grid, Image, Icon } from 'semantic-ui-react'
-import CustomRingLoader from './CustomRingLoader'
-import Videos from './Videos'
+import React, { Fragment, Component } from 'react'
+import { Header, Segment, Grid, Image, Icon, Form, Button } from 'semantic-ui-react'
 import ReactHtmlParser from 'react-html-parser'
 import moment from 'moment-timezone'
+import { connect } from 'react-redux'
+import CustomRingLoader from './CustomRingLoader'
+import Videos from './Videos'
+import { postEventLink } from '../actions/postEventLinkAction'
+import SingleFieldForm from './SingleFieldForm'
```

Comment:
hey @jmpainter thanks for following my suggestion to make this into a child functional component!
Am I right to assume that you are calling it `SingleFieldForm` with the idea that it can be reused elsewhere in the code base?
I saw that you are passing down the label and placeholder, so you could use this same component for say sharing the `YouTube` links :)

Reply:
Yes, my idea was that SingleFieldForm could be used in other places. In this situation I thought this approach needed review and I knew that it broke tests, but I wanted it to be looked at before committing the time to write tests. Would there be a better way to handle this kind of situation?

Reply:
I was actually saying that as a positive! I like that you are thinking ahead!! great job!!
As you said, we need to have tests to get it to pass the checks, but really nice start. I'll pull it down and see if I can get the back end ready